src/layers/domain/response/response_models.py

```diff
@@ -0,0 +1,13 @@
+from typing import Generic, List, TypeVar
+
+from domain.core.aggregate_root import AggregateRoot
+
+T = TypeVar("T", bound=AggregateRoot)
+
+
+class SearchResponse(Generic[T], AggregateRoot):
+    results: List[T]
+
+    @classmethod
+    def from_models(cls, models: List[T]) -> "SearchResponse[T]":
```

Comment:
what's the advantage of the method `from_models` over the standard constructor `SearchResponse(results=models)`